### ISSUE TITLE
release v3.5.0

### ISSUE THREAD

**Issue by edwardhartnett** [NOAA-EMC/wgrib2#289]
Can we release v3.5.0?

**Comment by edwardhartnett** [comment]
How close are we to a release?

People are waiting for the wgrib utility update, so let's get that out there...

**Comment by webisu** [comment]
Issues that I am looking at
   update ectables (update ncep tables too)
  fix complex-3 packing that that increase the nbits to 26
are not issues that need to be fixed immediately and will not affect NCEP
use.

The current build will not automatically build a libwgrib2.so that is python
compatible.  I was able to build a python libwgrib2.so by taking all the *.a
libraries but that is a manual process.  So 3.50 doesn't have to ability
to generate a python libwgrib2.so.

**Comment by edwardhartnett** [comment]
Did 3.4.0 have the ability to generate a python libwgrib2.so? Or is this a new capability?

**Comment by webisu** [comment]
Python support was added to wgrib2 v3.0.0.  The github version never had
the ability to generate a python compatible *.so.  Eventually it should be
added as I have seen "tutorials" on how to use wgrib2/python to process
NWS forecasts on AWS.  So 3.5.0 can go forward but the issue should
be addressed in the future.  (I tried when I added the shared library
support
but I didn't see any CMake/system independant way.)

**Comment by AlysonStahl-NOAA** [comment]
My main goal was to at least finish the png decoding implementation. I came across some challenges with it, but the implementation is completed now (and it works). I just need some changes to the g2c library to go through before those changes can be merged, but that PR is hinged on some issues with spack.

I can go ahead and move remaining issues to the next release. If we can wait on those changes for png decoding I can put out that release as soon as possible.

**Comment by edwardhartnett** [comment]
OK, WRT the python, I have added #295 , continue discussions there.

@webisu I've assigned it to the next release after this one, i.e. 3.6.0. Let's get 3.5.0 out with wgrib and then we can do the 3.6.0 release as soon as you're ready with the python changes. 

WRT the PNG issues, let's bump them to the next release if the current wgrib2 implementation meets those needs.

It's always hard to let go of code, but remember, there's always the next release to make it better. ;-) Let's get 3.5.0 out and move on to 3.6.0.
